final_task/rss_reader/requirements.txt

```diff
@@ -1,14 +1,4 @@
-astroid==2.3.3
 beautifulsoup4==4.8.1
 bs4==0.0.1
```

Comment:
A question:

What is the difference between bs4 and beautifulsoup4? Do you really need both of them?

Reply:
My bad. It seems when you install like "pip install bs4" it installs beautifulsoup4 and this dummy package. I will remove it.